server/clip_server/model/openclip_model.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, name: str, device: str = 'cpu', jit: bool = False, **kwargs):
         if model_url:
             model_path = download_model(model_url, md5sum=md5sum)
             self._model = load_openai_model(model_path, device=device, jit=jit)
-            self._model_name = name
+            self._model_name = name.split('::')[0]
```

Comment:
@numb3r3 Check this

Reply:
> @numb3r3 Check this

should be split[0], I had fixed it in my pr

Reply:
I think the problem caused here can be addressed by another PR. So let's revert this change in this PR.

Reply:
we need this to pass ci